src/Dialog/DialogBottomSheet.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import {render} from '@testing-library/react'
+import {Dialog} from './Dialog'
+import MatchMediaMock from 'jest-matchmedia-mock'
+import {behavesAsComponent, checkExports, renderStyles, checkStoriesForAxeViolations} from '../utils/testing'
+import {mediaQueries} from '../utils/layout'
+
+let matchMedia: MatchMediaMock
+
+describe('Dialog', () => {
+  beforeAll(() => {
+    matchMedia = new MatchMediaMock()
+  })
+
+  afterAll(() => {
```

Comment:
If you plan to use `matchMedia` to test things at different media queries, you probably want to change this to `afterEach`